src/orchestrator-tracing/src/lib.rs

```diff
@@ -178,6 +182,10 @@ pub struct TracingCliArgs {
     /// Sentry data source to submit events and exceptions (e.g. panics) to.
     #[clap(long, env = "SENTRY_DSN")]
     pub sentry_dsn: Option<String>,
+
+    /// Format to use for writing log messages to stderr.
+    #[clap(long, env = "STDERR_LOG_FORMAT", default_value = "text")]
+    pub stderr_log_format: TraceStderrFormat,
```

Comment:
We use "log" in the other arguments to mean "stderr log", so for consistency with the existing arguments this should just be called "--log-format".

```suggestion
    /// Format to use for writing log messages to stderr.
    #[clap(long, env = "LOG_FORMAT", default_value = "text")]
    pub log_format: TraceStderrFormat,
```